contracts/SpotV1.sol

```diff
@@ -123,7 +115,7 @@ contract SpotV1 is ERC20Spot {
 
         _mint(ownerId, amountToMint);
 
-        _returnRemainder(swap0.path[0]);
+        // _returnRemainder(swap0.path[0]);
```

Comment:
WARNING: tokens may be forever locked. Consider activate _returnRemainder, or at least add "recoverERC20" function. In the current implementation you may call "callAny" with calldata to IERC20.transfer but it's not intuitive.